jobs/build-node-image.Jenkinsfile

```diff
@@ -104,27 +104,38 @@ lock(resource: "build-node-image") {
                                                 "--label=org.opencontainers.image.version=${container_registry_staging_image_tag}.${timestamp}"])
             }
         }
-        stage('Build Extensions Image') {
-            withCredentials([file(credentialsId: 'oscontainer-push-registry-secret', variable: 'REGISTRY_AUTH_FILE')]) {
-                // Use the node image as from
-                def build_from = container_registry_staging_manifest
-                pipeutils.build_and_push_image(arches: arches,
-                                               src_commit: commit,
-                                               src_url: src_config_url,
-                                               staging_repository: container_registry_staging_repo,
-                                               image_tag_staging: "${container_registry_staging_image_tag}-extensions",
-                                               manifest_tag_staging: "${container_registry_staging_manifest_tag}-extensions",
-                                               secret: "id=yumrepos,src=${yumrepos_file}", // notsecret (for secret scanners)
-                                               from: build_from,
-                                               extra_build_args: ["--security-opt label=disable", "--mount-host-ca-certs",
-                                                                  "--git-containerfile", "extensions/Dockerfile", "--force"])
-            }
-        }
+        // stage('Build Extensions Image') {
```

Comment:
we want to uncomment this function back when we open a final PR?

Reply:
it's in a DNM commit :)